src/app/ui/hud/vitals.js

```diff
@@ -81,23 +75,23 @@ module.exports = class Vitals {
 
   static show(attrs) {
     const bar_klass =
-      attrs.id == "encumlevel" ||
-      attrs.id == "mindState" ||
-      attrs.id == "nextLvlPB"
+      attrs.id == "encumlevel" || attrs.id == "mindState"
         ? Progress.classify_down(attrs.percent)
         : Progress.classify(attrs.percent)
 
-    return m(`li#vitals-${attrs.id}`, { key: attrs.id }, [
-      m(
-        `.value.${attrs.text.length > 1 ? "" : "center"}`,
-        attrs.text.map(span),
-        [m("span.max.2", attrs.max <= 0 ? "" : attrs.max)]
-      ),
-      m(
-        `.bar.${bar_klass}`,
-        Lens.put({}, "style.width", attrs.percent + "%")
-      ),
-    ])
+    const [text, value] = attrs.text
+
+    return m(
+      `li#vitals-${attrs.id}.${bar_klass}.vital`,
+      { key: attrs.id },
+      [
+        text && span(text, ".label"),
+        value && m("span.value", value),
+        isNaN(attrs.max)
+          ? void 0
+          : m("span.max", attrs.max),
+      ]
+    )
   }
```

Comment:
This the big Nuking of the Progress Bar. 

But there are a few other progress bars, and I'd think we should nuke them in the spirit of better threshold notifiers in the same spirit as the vials. 

- **Active Spell bars**. The bars are based on ~4 hours flat, which doesn't make sense for a lot of spells. Just showing the time remaining is probably the best bet until we can do something smarter like per-spell timings.
- **Stance bar.** Makes a little more sense, but it might be weird to be the only vital with a progress bar. Certainly the word of stance (plus the %) is enough information.

Reply:
I'm fine with getting rid of them all, I just wanted to push my code for people to see first

Reply:
Late to the party, but: Personally, I feel like the bars do a pretty good job of communicating the current deficit for a given vital. Visually speaking, It's a much more immediately obvious to me than having to compare a "current value/max value" set of numbers. I suppose we can add some color shifting to the labels, etc. but... Anyway. guess this ship has sailed.

Reply:
The bars were also some of the only non-text-based UI in the window, which was nice.

Reply:
The issue  with the old pattern was if you're at 4/170 hp, the old UI would render at 1% width, which is barely visible.

This is the exact opposite of what needs to happen it should become more visible, now the entire `<li>` element can be styled according to the current threshold, and thresholds can be potentially configured.  Missing 30% of health on a warrior is not as interesting at 30% of health on a wizard for instance.

```
<li id="vitals-mana" class="low vital">
           <span class="label">mana</span>
           <span class="value">62</span>
           <span class="max">266</span>
</li>
```

They can still very much have non-text UI components if a skin wants to define glyphs for `.low`, `.medium`, and `.high`

Reply:
I was a bit unsure about the removal of bars too, but I think it's a strong point that they often don't accomplish what they ought to. I'm trying to embrace that as I jiggle with a theme of mine by calling attention to vitals with the classes when they are in a low state.

<img width="261" alt="Screen Shot 2020-07-26 at 2 21 40 PM" src="https://user-images.githubusercontent.com/69156/88489846-dd44ec80-cf4b-11ea-94c6-c21d49f20744.png">